### PR TITLE
URL code style consistency

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Fix URL style inconsistencies
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-416575](https://oktainc.atlassian.net/browse/OKTA-416575)

### CODE REVIEW

**Review by brentschaus-okta — APPROVED** [review]
Looks good to me. I did a quick "find" in the docs project and wasn't able to find any URLs still using the incorrect format.

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/go/accesstoken.md [hunk 1]
```diff
@@ -15,23 +15,23 @@ import github.com/okta/okta-jwt-verifier-golang
 
 toValidate := map[string]string{}
 toValidate["aud"] = "api://default"
-toValidate["cid"] = "{CLIENT_ID}"
+toValidate["cid"] = "${CLIENT_ID}"
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/go/accesstoken.md:18]:
```suggestion
toValidate["cid"] = "${clientId}"
```

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/go/accesstoken.md [hunk 2]
```diff
@@ -15,23 +15,23 @@ import github.com/okta/okta-jwt-verifier-golang
...
 
 jwtVerifierSetup := jwtverifier.JwtVerifier{
-        Issuer: "{ISSUER}",
+        Issuer: "${ISSUER}",
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/go/accesstoken.md:21]:
```suggestion
        Issuer: "${issuer}",
```

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/go/accesstoken.md [hunk 3]
```diff
@@ -15,23 +15,23 @@ import github.com/okta/okta-jwt-verifier-golang
...
-token, err := verifier.VerifyAccessToken("{JWT}")
+token, err := verifier.VerifyAccessToken("${JWT}")
 ```
 
 You may need to adjust your clock skew leeway. We default to a `PT2M` clock skew adjustment in our validation. You can adjust this to your needs by setting the `SetLeeway` in seconds after getting a new instance of the verifier:
 
 ```go
 jwtVerifierSetup := JwtVerifier{
-        Issuer: "{ISSUER}",
+        Issuer: "${ISSUER}",
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/overview/go/accesstoken.md:34]:
```suggestion
        Issuer: "${issuer}",
```

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 4]
```diff
@@ -588,7 +588,7 @@ curl -v -X POST \
   },
   "_links": {
     "authorize": {
-      "href": "http://{yourOktaDomain}/oauth2/v1/authorize?idp=0oa18hsHsG3boVejU0g4&client_id={clientId}&response_type={responseType}&response_mode={responseMode}&scope={scopes}&redirect_uri={redirectUri}&state={state}&nonce={nonce}",
+      "href": "http://${yourOktaDomain}/oauth2/v1/authorize?idp=0oa18hsHsG3boVejU0g4&client_id={clientId}&response_type={responseType}&response_mode={responseMode}&scope={scopes}&redirect_uri={redirectUri}&state={state}&nonce={nonce}",
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:591]:
```suggestion
      "href": "http://${yourOktaDomain}/oauth2/v1/authorize?idp=0oa18hsHsG3boVejU0g4&client_id=${clientId}&response_type=${responseType}&response_mode=${responseMode}&scope=${scopes}&redirect_uri=${redirectUri}&state=${state}&nonce=${nonce}",
```

**Review by vanngo-okta — APPROVED** [review]
A few nits. Otherwise, LGTM!

**Comment by christinechoy-okta** [comment]
All of Van's review comments are resolved in https://github.com/okta/okta-developer-docs/pull/2486

**Pull request merged**